src/django/api/migrations/0179_introduce_enable_v1_claims_flow.py

```
@@ -0,0 +1,28 @@
+from django.db import migrations
+
+
+def create_enable_v1_claims_flow_switch(apps, schema_editor):
```

Ruff (0.12.2)

Unused function argument: `schema_editor`

(ARG001)